planet/clients/subscriptions.py

```diff
@@ -6,6 +6,9 @@
 from planet.exceptions import APIError, ClientError
 from planet.http import Session
 from planet.models import Paged
+from ..constants import PLANET_BASE_URL
+
+BASE_URL = f'{PLANET_BASE_URL}/subscriptions/v1/'
```

Comment:
Thank you for moving this base URL here ([related thread](https://github.com/planetlabs/planet-client-python/pull/701#discussion_r986184740)). 👍

Reply:
great memory! Yep, this has been on my mind since that discussion and I'm glad to have it resolved.